test/functions/generic/poi/issue-19198m.chpl

```diff
@@ -0,0 +1,28 @@
+module G {
+  proc genericfunc(param p) {
+    helper1();
+  }
+}
+
+module M {
+  use G;
+  proc mCallFunc() {
+    writeln("mCallFunc");
+    genericfunc(1);
+  }
+  proc helper1() { writeln("M.helper1"); }
+}
+
+module N {
+  use G,M;
+  proc helper1() { writeln("N.helper1"); }
+  proc helper1() { writeln("N.helper1 mark two"); }
```

Comment:
Is the reason for two identical overloads here just to reinforce the point (beyond the printed message) that neither of these are called / since they're not considered, they don't result in an ambiguity?

Reply:
IIRC I wanted to add a variant of the test with did the part that would run, since the usual variant results in a compilation error. I also think it is nice to demonstrate the compiler isn't getting confused and calling the wrong `helper1` since the issue these tests are for is about confusion with POI. The two identical overloads existed in the original of this test (issue-19198.chpl).